electrum_nmc/electrum/commands.py

```diff
@@ -1502,6 +1502,7 @@ def eval_bool(x: str) -> bool:
     'to_height':   (None, "Only show transactions that confirmed before given block height"),
     'destination': (None, "Namecoin address, contact or alias"),
     'amount':      (None, "Amount to be sent (in NMC). Type \'!\' to send the maximum available."),
+    'outputs':     (None, "Coin outputs to add to a transaction in addition to a name operation."),
```

Comment:
Can you reword "Coin outputs" to "Currency outputs"?  The term "coin" is a bit ambiguous since Electrum often uses the word "coin" to refer to any output (including name outputs).

Reply:
Makes sense, done.